bitnami/nessie/templates/_helpers.tpl

```diff
@@ -0,0 +1,323 @@
+  command:
+    - /bin/bash
+  args:
+    - -ec
+    - |
+      #!/bin/bash
+      mkdir -p {{ .Values.persistence.mountPath }}
+      chown {{ .Values.volumePermissions.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }} {{ .Values.persistence.mountPath }}
+      find {{ .Values.persistence.mountPath }} -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R {{ .Values.volumePermissions.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }}
+  {{- if .Values.volumePermissions.containerSecurityContext.enabled }}
+  securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.volumePermissions.containerSecurityContext "context" $) | nindent 4 }}
+  {{- end }}
+  {{- if .Values.volumePermissions.resources }}
+  resources: {{- toYaml .Values.volumePermissions.resources | nindent 4 }}
+  {{- else if ne .Values.volumePermissions.resourcesPreset "none" }}
+  resources: {{- include "common.resources.preset" (dict "type" .Values.volumePermissions.resourcesPreset) | nindent 4 }}
+  {{- end }}
+  volumeMounts:
+    - name: data
+      mountPath: {{ .Values.persistence.mountPath }}
+{{- end -}}
+
+{{- define "nessie.waitForDBInitContainer" -}}
+# We need to wait for the postgresql database to be ready in order to start with Nessie.
+- name: wait-for-db
+  image: {{ template "nessie.psql.image" . }}
+  imagePullPolicy: {{ .Values.psqlImage.pullPolicy }}
+  {{- if .Values.containerSecurityContext.enabled }}
+  securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.containerSecurityContext "context" $) | nindent 4 }}
+  {{- end }}
+  command:
+    - bash
+    - -ec
+    - |
+      #!/bin/bash
+
+      set -o errexit
+      set -o nounset
+      set -o pipefail
+
+      . /opt/bitnami/scripts/liblog.sh
+      . /opt/bitnami/scripts/libvalidations.sh
+      . /opt/bitnami/scripts/libpostgresql.sh
+      . /opt/bitnami/scripts/postgresql-env.sh
+
+      {{- if .Values.usePasswordFile }}
+      export DATABASE_PASSWORD="$(< "/bitnami/nessie/secrets/database/QUARKUS_DATASOURCE_POSTGRESQL_PASSWORD")"
+      {{- end }}
+      info "Waiting for host $DATABASE_HOST"
+      psql_is_ready() {
+          if ! PGCONNECT_TIMEOUT="5" PGPASSWORD="$DATABASE_PASSWORD" psql -U "$DATABASE_USER" -d "$DATABASE_NAME" -h "$DATABASE_HOST" -p "$DATABASE_PORT_NUMBER" -c "SELECT 1"; then
+             return 1
+          fi
+          return 0
+      }
+      if ! retry_while "debug_execute psql_is_ready"; then
+          error "Database not ready"
+          exit 1
+      fi
+      info "Database is ready"
+  {{- if .Values.resources }}
+  resources: {{- toYaml .Values.resources | nindent 4 }}
+  {{- else if ne .Values.resourcesPreset "none" }}
+  resources: {{- include "common.resources.preset" (dict "type" .Values.resourcesPreset) | nindent 4 }}
+  {{- end }}
```

Comment:
I don't like the idea of setting the same resources used for nesie on this init-container

Reply:
Created a new waitContainer section